CHANGELOG.md

```diff
@@ -6,6 +6,128 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0] - June, 27, 2022
+
+### Design Tokens
+#### Breaking Changes
+#### Changed
+- [Typography] Typography tokens to match hel.fi new visual identity
+
+#### Fixed
+- [Colour] The grayscale tokens hex value to match the percentage name
+
+### Core
```

Comment:
This is missing a mention about heading helper classes. I assume they are part of Core.

Reply:
- [x] add missing mention